homeassistant/components/motionblinds_ble/entity.py

```diff
@@ -0,0 +1,53 @@
+"""Base entity for the Motionblinds BLE integration."""
+
+import logging
+
+from motionblindsble.const import MotionBlindType
+from motionblindsble.device import MotionDevice
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_ADDRESS
+from homeassistant.helpers.device_registry import CONNECTION_BLUETOOTH, DeviceInfo
+from homeassistant.helpers.entity import Entity, EntityDescription
+
+from .const import CONF_BLIND_TYPE, CONF_MAC_CODE, MANUFACTURER
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class MotionblindsBLEEntity(Entity):
+    """Base class for Motionblinds BLE entities."""
+
+    _attr_has_entity_name = True
+    _attr_should_poll = False
+
+    _device: MotionDevice
+    config_entry: ConfigEntry
+
+    def __init__(
+        self,
+        device: MotionDevice,
+        entry: ConfigEntry,
```

Comment:
Please match the naming to what it's called on the object or switch the object

Reply:
I assume you mean for device and entry? I changed all variables to use only `device` and `entry` instead of `_device` and `config_entry`

Reply:
`entry` and `config_entry` was what I was concerned about. But that works as well